### ISSUE TITLE
Optional Nullability

### ISSUE THREAD

**Issue by cesarferreira** [FlutterGen/flutter_gen#108]
First of all, amazing tool! Use it on the daily!

Is there a way to tell the tool not to generate dart 2.12 optional fields?
Many of my projects are still on non null-sound libraries, it's quite tedious to go and delete all the `?` every time the tool runs

**Comment by lcdsmao** [comment]
Hi @cesarferreira.
Sorry for the lack of documentation.
Could you try to modify your `pubspec.yaml`:
```yaml
flutter_gen:
  null_safety: false
```
And then run fluttergen again?

**Comment by cesarferreira** [comment]
It worked! but this line at the top makes it complain that im not null sound

```
// @dart = 2.10
```

![image](https://user-images.githubusercontent.com/277426/119245939-19011080-bb75-11eb-8549-4a52f5cc2e2b.png)

**Comment by lcdsmao** [comment]
@cesarferreira 

>It worked! but this line at the top makes it complain that im not null sound

Did you mean this?
<img width="677" alt="Screen Shot 2021-05-23 at 11 39 27" src="https://user-images.githubusercontent.com/13031690/119246310-89328500-bbbb-11eb-9f8f-b1311c962b78.png">

**Comment by cesarferreira** [comment]
My min sdk is 2.7.0 and even tho the fields are not optional anymore, the line at the top that says

```
// @dart = 2.10
```

turns the whole document red, i can easily fix it by simply removing that // dart = 2.10 line but i need to go there every time, is there a way to turn it off? to not have it being pasted at the beginning of the file?

**Comment by lcdsmao** [comment]
@cesarferreira 
Thanks for the information about your project.
After discussion with @wasabeef, we are deciding to not support the project with dart sdk < 2.12.
The version comment `@dart=2.10 ` and pubspec key `null_safety: false` is aim to support the project that cannot fully migrate to null-safety (dart >= 2.12).
So if you want to use flutter_gen with your project, please try the old version like 1.31.
Thanks.
